libmscore/figuredbass.cpp

```diff
@@ -17,10 +17,13 @@
 #include "system.h"
 #include "segment.h"
 #include "chord.h"
+#include "staff.h"
 #include "rest.h"
 #include "score.h"
 #include "sym.h"
 #include "xml.h"
+#include "utils.h"
+#include "mscore/scoreview.h"
```

Comment:
We didn't include any code. 
I just click hover on a line number and a (+) appear to comment this line.
![github](https://cloud.githubusercontent.com/assets/234271/8616258/0a406ac0-26fa-11e5-989e-a7780fb12c1d.gif)